<launch>

    <group ns ="robot_1">
        <node name="wiggle1" pkg="week3" type="wiggler.py" />
    </group>

    <group ns ="robot_2">
        <node name="wiggle2" pkg="week3" type="wiggler.py" />
    </group>

    <group ns ="robot_3">
        <node name="wiggle3" pkg="week3" type="wiggler.py" />
    </group>

</launch>